src/compas_viewer/scene/scene.py

```diff
@@ -60,6 +62,15 @@ class ViewerScene(Scene):
     context : str, optional
         The context of the scene.
 
+    Attributes
+    ----------
+    observers : set
+        Set of unique observer objects.
```

Comment:
are these objects that observe, or objects that are being observed?

what makes something an "observer"?

is there a base class somewhere?

Reply:
I updated the observer class in  PR #181 maybe I remove from here, so is less confusing